Simplenote/src/main/java/com/automattic/simplenote/networking/SimpleHttp.kt

```diff
@@ -0,0 +1,57 @@
+package com.automattic.simplenote.networking
+
+import okhttp3.HttpUrl
+import okhttp3.MediaType
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody
+import okhttp3.Response
+import org.json.JSONException
+import org.json.JSONObject
+import java.util.concurrent.TimeUnit
+
+object SimpleHttp {
```

Comment:
Just a heads up. While trying to add the unit tests I began to run into problems. Firstly, mocking static methods is annoying due to how behind we are with library updates. So I will have to make it a regular class. I will move `OkHttp` into a singleton instance using hilt so it can be injected into an `SimpleHttp` instance. Then we'll also inject `SimpleHttp`. This will make unit testing easier. I'll do it monday morning.

There is the other issue of passing `Simplenote`, which I cannot mock with the current version of mockito. I will try and also inject that in somehow through an interface. I have some thoughts. Either way, with the exception of those gotchas feel free to review the rest of it, which I will likely not change.